src/ts/component/menu/dataview/object/list.tsx

```diff
@@ -67,7 +68,14 @@ const MenuDataviewObjectList = observer(class MenuDataviewObjectList extends Rea
 						<div className="inner" />
 					</div>
 				);
-			} else if (item.id == 'add') {
+			} else
+			if (item.isEmpty) {
```

Comment:
why did you add this logic? there is already empty check:
`{!items.length && !loading ? (
					<EmptySearch text={filter ? UtilCommon.sprintf(translate('popupSearchEmptyFilter'), filter) : translate('popupSearchEmpty')} />
				) : ''}`